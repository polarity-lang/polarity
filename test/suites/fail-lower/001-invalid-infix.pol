data Foo { C(x: Type) }

infix x + y := C(z)
