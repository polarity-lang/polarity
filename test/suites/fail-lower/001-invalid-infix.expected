L-001

  × Undefined identifier z
   ╭─[001-invalid-infix.pol:3:18]
 2 │ 
 3 │ infix x + y := D(z)
   ·                  ─
   ╰────
